en/sql/function/json_fn.rst

```diff
@@ -0,0 +1,285 @@
+:meta-keywords: cubrid json, json functions, database json
+:meta-description: CUBRID functions that create, query and modify JSON data.
+
+:tocdepth: 3
+
+*********************************
+JSON functions
+*********************************
+
+.. contents::
+
+JSON_ARRAY
+===================================
+
+.. function:: JSON_ARRAY ([val1 [, val2] ...])
+
+  The **JSON_ARRAY** function returns a json array containing the given list (possibly empty) of values.
+
+.. code-block:: sql
+
+    SELECT JSON_ARRAY();
+::
+
+      json_array()
+    ======================
+      []
+
+.. code-block:: sql
+
+    SELECT JSON_ARRAY(1, '1', json '{"a":4}', json '[1,2,3]');
+::
+
+      json_array(1, '1', json '{"a":4}', json '[1,2,3]')
+    ======================
+      [1,"1",{"a":4},[1,2,3]]
+
+JSON_OBJECT
+===================================
+
+.. function:: JSON_OBJECT ([key1, val1[ , key2, val2] ...])
+
+  The **JSON_OBJECT** function returns a json object containing the given list (possibly empty) of key-value pairs.
+
+.. code-block:: sql
+
+    SELECT JSON_OBJECT();
+::
+
+      json_object()
+    ======================
+      {}
+
+.. code-block:: sql
+
+    SELECT JSON_OBJECT('a', 1, 'b', '1', 'c', json '{"a":4}', 'd', json '[1,2,3]');
+::
+
+      json_object('a', 1, 'b', '1', 'c', json '{"a":4}', 'd', json '[1,2,3]')
+    ======================
+      {"a":1,"b":"1","c":{"a":4},"d":[1,2,3]}
+
+JSON_KEYS
+===================================
+
+.. function:: JSON_KEYS (json_doc [ , json path])
+
+  The **JSON _KEYS** function returns a json array of all the object keys of the json object at the given path. Json null is returned if the path addresses a json element that is not a json object.
+  If no argument is given, the keys are gathered from the root path ('$'). Returns NULL if json_doc argument is NULL.
+
```

Comment:
Add a sentence to explain what happens if path does not exist.

I think it would look better if lines are organized like this:
  first list - general description
  second line - explains the json path argument (3 sentences)
  last line - returns null on null argument